README.md

```diff
@@ -18,106 +23,143 @@ Installation
 ------------
 
 Add the following line to you `composer.json` file under `require`.
-```json
```

Comment:
As you mentioned on the last pull request I used to use the code hints. I dod not touch this line why that is gone ? 0.o